src/core/src/bootstrap/EnvLayer.py

```diff
@@ -284,24 +284,29 @@ def open(self, file_path, mode):
                     if i < Constants.MAX_FILE_OPERATION_RETRY_COUNT:
                         time.sleep(i + 1)
                     else:
-                        raise Exception("Unable to open {0} (retries exhausted). Error: {1}.".format(str(real_path), repr(error)))
+                        error_message = "Unable to open file (retries exhausted). [File={0}][Error={1}][RaiseIfNotFound={2}].".format(str(real_path), repr(error), str(raise_if_not_found))
+                        print(error_message)
```

Comment:
Shouldn't this only be in the 'return None' case?

Reply:
Resolved